src/Http/Routing/src/RoutingMetrics.cs

```diff
@@ -10,33 +10,30 @@ internal sealed class RoutingMetrics
     public const string MeterName = "Microsoft.AspNetCore.Routing";
 
     private readonly Meter _meter;
-    private readonly Counter<long> _matchSuccessCounter;
-    private readonly Counter<long> _matchFailureCounter;
+    private readonly Counter<long> _matchAttemptsCounter;
 
     public RoutingMetrics(IMeterFactory meterFactory)
     {
         _meter = meterFactory.Create(MeterName);
 
-        _matchSuccessCounter = _meter.CreateCounter<long>(
-           "routing-match-success",
-            description: "Number of requests that successfully matched to an endpoint.");
-
-        _matchFailureCounter = _meter.CreateCounter<long>(
-           "routing-match-failure",
-            description: "Number of requests that failed to match to an endpoint. An unmatched request may be handled by later middleware, such as the static files or authentication middleware.");
+        _matchAttemptsCounter = _meter.CreateCounter<long>(
```

Comment:
I think the unit should be "{match_attempt}" if we are following the naming of the instrument, but @lmolkova's PR currently has it as "{match}". Do folks agree that it should be {match_attempt} and we should update @lmolkova's PR?